woocommerce/woocommerce-gutenberg-products-block
# Add new icon for mini cart block

Updates the inserter icon for the Mini Cart block to better differentiate between the Mini Cart and the Cart blocks.





Fixes #6617 




### Screenshots



| Before | After |
| ------ | ----- |
|  ![CleanShot 2022-07-28 at 14 57 58](https://user-images.githubusercontent.com/481776/181622493-d618649c-e9ee-4649-9e10-a2aed6737fc5.png)  |  ![CleanShot 2022-07-28 at 15 31 41](https://user-images.githubusercontent.com/481776/181622516-4bade22f-944a-488e-aacc-6094afa97efe.png)  |

### Testing



1. Make sure you have a blocks theme active (like Twenty Twenty-Two).
2. Navigate to **Appearance > Editor (Beta)** and add a Mini Cart block somewhere in your site.
3. Select the Mini Cart block in the editor and confirm the icon in the inserter and toolbar reflect the above changes.

### WooCommerce Visibility





### Changelog

> Update: New block icon for the Mini Cart block.
